### PR TITLE
Add build time operators

### PR DESCRIPTION
**Pull request by JohannMoller**
Closes #ISSUE_NUMBER

## Checklist

- [ ] Pull request is made to the "develop" branch
- [ ] Tests added
- [ ] Documentation added/updated
- [x] Code has been formatted with Prettier
- [x] Edits from maintainers are allowed

### CODE REVIEW

File: packages/operators/src/nodeParser.js [hunk 1]
```diff
@@ -52,33 +54,35 @@ class NodeParser {
...
-          return null;
-        }
+      if (!type.isObject(value) || Object.keys(value).length !== 1) return value;
+
+      const key = Object.keys(value)[0];
+      if (!key.startsWith(operatorPrefix)) return value;
+
+      const [op, methodName] = `_${key.substring(operatorPrefix.length)}`.split('.');
+      if (type.isUndefined(operators[op])) return value;
+      console.log(op);
```
Comment (Gervwyk) [inline packages/operators/src/nodeParser.js:64]: Remove log.

File: packages/plugins/operators/operators-js/src/operatorsBuild.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import _product from './operators/shared/product.js';
+import _random from './operators/shared/random.js';
+import _regex from './operators/shared/regex.js';
+import _string from './operators/shared/string.js';
+import _subtract from './operators/shared/subtract.js';
+import _sum from './operators/shared/sum.js';
+import _switch from './operators/shared/switch.js';
+import _type from './operators/shared/type.js';
+import _uri from './operators/shared/uri.js';
+import _user from './operators/shared/user.js';
```
Comment (Gervwyk) [inline packages/plugins/operators/operators-js/src/operatorsBuild.js:49]: Can _user be a build operator?

Comment (JohannMoller) [reply packages/plugins/operators/operators-js/src/operatorsBuild.js:49]: No, I don't think so. I will take it out

**Comment by Gervwyk** [comment]
Looks great @JohannMoller 💫 Please add tests, and docs for `_env` and `_build`. Thanks.

File: packages/build/src/build/buildRefs/evaluateBuildOperators.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { NodeParser } from '@lowdefy/operators';
+import operators from '@lowdefy/operators-js/operators/build';
+
+async function evaluateBuildOperators({ context, input, refDef }) {
+  const operatorsParser = new NodeParser({
+    env: process.env,
+    operators,
+  });
+
+  await operatorsParser.init();
```
Comment (Gervwyk) [inline packages/build/src/build/buildRefs/evaluateBuildOperators.js:26]: We are not doing init and async anymore for operators, so this needs to change.

**Review by Gervwyk — APPROVED** [review]

**Pull request merged**